app/app.go

```diff
@@ -708,6 +735,25 @@ func New(
 
 	app.SetEndBlocker(app.EndBlocker)
 
+	planName := "v0.8.0"
+	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
```

Comment:
should it set the minimum timeout duration?

Reply:
if it's from that chain-main value, I guess it should be ok... it was based on a discussion @calvinaco @devashishdxt decided referencing the relayer time backlog from the past

Reply:
> the default value is 1hour, looks good? I'm reenabling the upgrade test case to assert this.

done, added the upgrade integration test.